### DISCUSSION TITLE
I am new to git. Please help in making my first commit

### DISCUSSION THREAD

**Discussion by ghost** [community/community#103165] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hey can anyone pls help in git

**Comment by Mujster** [comment, marked as answer]
git init //to initialize git folder
git add . //to add all untracked files
git commit -m "[your commit message]"
git branch //to check current branch
git branch -M [branch name] //to switch to that branch
git status //check status of the current git
git remote add origin [ssh] //to add remote origin
git push origin [branch name] to push to a repo on github
